src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/configuration/configuration.component.ts

```diff
@@ -46,9 +49,9 @@ export class ConfigurationComponent implements OnInit {
     }
   ];
 
-  constructor(private configurationService: ConfigurationService) {}
-
-  ngOnInit() {
+  constructor(
+    private configurationService: ConfigurationService,
+  ) {
     this.columns = [
```

Comment:
Some columns will need the `wrap` css class.

Reply:
That means I will have to add ```{ flexGrow: 2, prop: 'desc', name: 'Description', class: 'wrap' }``` for example?

Reply:
I think you need to use `cellClass`.